portfolio/src/main/webapp/index.html

```diff
@@ -48,9 +46,10 @@ <h1>Comments:</h1>
 
       <input type="submit" />
       </form>
-
-      <a id="loginLink" href = "/_ah/login?continue=%2F" title="Login here">Login</a>
-      <a id="logoutLink" href = "/_ah/logout?continue=%2F" title="Logout here">Logout</a>
+      
+      <a id="changeNicknameLink" href="/nickname" title = "Change nickname">Change your nickname here</a>
```

Comment:
Where in your java server are requests to "/nickname" handled?

Reply:
Requests to "/nickname" are handled in NicknameServlet.java

Reply:
Ah gotcha, this is following up on a previous PR.

Fine for now, but please separate unrelated changes in their own PRs in the future.